src/SDK/Logs/LoggerProviderBuilder.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Logs;
+
+use OpenTelemetry\SDK\Common\Attribute\Attributes;
+use OpenTelemetry\SDK\Common\Instrumentation\InstrumentationScopeFactory;
+use OpenTelemetry\SDK\Logs\Processor\NoopLogsProcessor;
+use OpenTelemetry\SDK\Resource\ResourceInfo;
+
+class LoggerProviderBuilder
+{
+    private ?LogRecordProcessorInterface $processor = null;
+    private ?ResourceInfo $resource = null;
+
+    public function addLogRecordProcessor(LogRecordProcessorInterface $processor): self
+    {
+        $this->processor = $processor;
```

Comment:
```suggestion
        $this->processors[] = $processor;
```

Reply:
I think you're right that we need to support multiple processors (the spec is clearer for logs, whereas in traces having more than one processor is optional). I'll change these methods to `set` for now, but create an issue to support multiple.